codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/Syntax.scala

```diff
@@ -15,7 +15,10 @@
  */
 
 package com.lightbend.akkasls.codegen
-
+/*
+ * Be mindful that `indent` adds to the immediate
+ * upper line (blank line) with the same indent.
+ */
```

Comment:
how can we improve it as such that we don't need a comment? 

Also, if we need such a comment, better to add it to the `indent` method.

Reply:
fixed now